internal/webapi/server.go

```diff
@@ -70,6 +71,10 @@ func (a *Api) AllStatistics(r *http.Request, l *slog.Logger) (*femto.Response[br
 	if err != nil {
 		return nil, err
 	}
+	if data == nil {
```

Comment:
this bit is the important bit for returning `[]` from allstats instead of `null` when the db is empty.